index.d.ts

```diff
@@ -216,27 +217,27 @@ declare namespace Emittery {
 	emitter.emit('end'); // TS compilation error
 	```
 	*/
-	class Typed<EventDataMap extends Events, EmptyEvents extends string = never> extends Emittery {
-		on<Name extends Extract<keyof EventDataMap, string>>(eventName: Name, listener: (eventData: EventDataMap[Name]) => void): Emittery.UnsubscribeFn;
+	class Typed<EventDataMap extends Events, EmptyEvents extends (string | symbol) = never> extends Emittery {
+		on<Name extends Extract<keyof EventDataMap, (string | symbol)>>(eventName: Name, listener: (eventData: EventDataMap[Name]) => void): Emittery.UnsubscribeFn;
 		on<Name extends EmptyEvents>(eventName: Name, listener: () => void): Emittery.UnsubscribeFn;
 
-		events<Name extends Extract<keyof EventDataMap, string>>(eventName: Name): AsyncIterableIterator<EventDataMap[Name]>;
+		events<Name extends Extract<keyof EventDataMap, (string | symbol)>>(eventName: Name): AsyncIterableIterator<EventDataMap[Name]>;
```

Comment:
Should `Extract<keyof EventDataMap, (string | symbol)>` be moved to a named `type` maybe? `EventName` or `BaseEventName`?